tests/data/syntax-error-in-prepared-statement.php

```diff
@@ -173,4 +173,30 @@ public function arrayParam(Connection $connection)
         $query = 'SELECT email FROM ada WHERE adaid IN (:adaids)';
         $connection->preparedQuery($query, ['adaids' => [1, 2, 3]]);
     }
+
+    public function noErrorInBug94(Connection $connection)
+    {
+        // XXX with proper sql parsing, we should better detect the placeholders and therefore could validate this query
+        $sql = "
+            INSERT IGNORE INTO `s_articles_supplier` (`id`, `name`, `img`, `link`, `changed`) VALUES (:supplierId, 'TestSupplier', '', '', '2019-12-09 10:42:10');
```

Comment:
the current placeholder parsing detects `:42` of the datetime-col as a placeholder